### ISSUE TITLE
progress DynamicDistortionCalibrator

### ISSUE THREAD

**Issue by ghost** [slinshady89/dynamic_distortion_calibrator#6]
Now that the other two classes are largely finished, we can start doing stuff here.

1. implement the save & load function for the raw distortion `saveRawDistortion` & `loadRawDistortion` so that we only have to run the areaDetector once, because it's so slow.

2. we can implement a way to find the middle point of the area and thus the offsets, so that'd be `calculateOffset`. Maybe we should add variables for the image's middlepoint to our cameraArea struct.

3. once 2. is done we can actually do `correctForOffset` and then we already have the distortions.

4. fit a model function

5. save and load functions for the model.

6. actual correction of the image. Maybe we can check whether we have the raw distortion or a model and make it so, that either can be used for correction.

**Comment by slinshady89** [comment]
the saving of offset will be difficult because then our scenario can't change. if we place the camera not like the last time it's worthless.
I think we only need to safe our matching matrices to undistort the image.
the resolution of the camera is fix and so the distortion should be too.

**Comment by ghost** [comment]
yeah, we don't need to save the offset, only the two match arrays. Everything else is variable.

**Comment by ghost** [comment]
Closing was accidental, shouldn't have written at 04:30 in the morning. Sorry all!

Saving and loading is still open
